examples/amphtml-ads/many-ads.host.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Many AMPHTML ads</title>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style>
+    body {
+      margin: 0;
+    }
+    h3 {
+      margin-left: 20px;
+    }
+    iframe {
+      width: calc(100vw - 2px);
+      height: 250px;
+      border: 1px solid lightgrey;
+    }
+    hr {
+      margin-top: 30px;
+      border-top:1px dashed #999;
+    }
+    a {
+      display: block;
+    }
+  </style>
+</head>
+<body>
+<h3>Image ad w/ Ad Choices</h3>
+<iframe
+    src="//ads.localhost:8000/examples/amphtml-ads/adchoices-1.a4a.html"
```

Comment:
Just question, is it intended to create a cross-domain iframe?

Reply:
yes. this is cross-domain inabox rendering